packages/scandipwa/src/component/PureForm/FieldFile/FieldFile.component.js

```diff
@@ -78,7 +78,7 @@ export class FieldFile extends PureComponent {
             setRef
         } = this.props;
 
-        const allowedFieldTypes = accept
+        const allowedFieldTypes = (accept || '')
```

Comment:
`accept` constant already have default assignment on line 76.
Do we really need a second one?